helm/charts/oathkeeper/charts/oathkeeper-master/templates/deployment.yaml

```diff
@@ -41,7 +39,7 @@ spec:
             - /manager
           args:
             - --metrics-addr=127.0.0.1:8080
-            - --rulesConfigmapName={{ .Values.configMapName }}
+            - --rulesConfigmapName={{ .Release.Name }}-rules
```

Comment:
I think this needs to be `oathkeeper.fullname` right?

Reply:
no, as this is a subchart, it inherits the name of its parent, so `fullname` produces `oathkeeper-oathkeeper-maester`

Reply:
After some thought I changes this to a smarter function